mmv1/products/compute/api.yaml

```diff
@@ -15647,6 +15647,99 @@ objects:
         required: true
         update_verb: :POST
         update_url: 'projects/{{project}}/regions/{{region}}/targetHttpsProxies/{{name}}/setUrlMap'
+  - !ruby/object:Api::Resource
+    name: 'RegionTargetTcpProxy'
+    kind: 'compute#targetTcpProxy'
+    base_url: projects/{{project}}/regions/{{region}}/targetTcpProxies
+    collection_url_key: 'items'
```

Comment:
This isn't used and can be removed

Reply:
collection_url_key is unused, so yeah you can remove it from those other resources

Reply:
Ok, I removed them as well. The resource does not support patching fields, so now it's recreate based (I left the update test in, setBackendService/setProxyHeader gets added to API at some point).